src/store/getters/state-getters.ts

```diff
@@ -506,6 +506,12 @@ export const hasContactInfoChanged = (state: StateIF): boolean => {
   )
 }
 
+/** The dynamic Office type based on the current filing. */
+export const officeType = (state: StateIF): OfficeTypes => {
+  if (isChangeFiling(state)) return OfficeTypes.BUSINESS_OFFICE
+  if (isAlterationFiling(state) || isCorrectionFiling(state)) return OfficeTypes.REGISTERED_OFFICE
+}
```

Comment:
I think this will return undefined if none of the other conditions are true, but it's better to return something anyway.